apps/mudita-center-e2e/src/page-objects/help.page.ts

```diff
@@ -7,17 +7,54 @@ import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 class HelpPage extends Page {
-  public get helpListElement(): ChainablePromiseElement<
+  public get listElement(): ChainablePromiseElement<
     Promise<WebdriverIO.Element>
   > {
     return $('[data-testid="help-component-question"]')
   }
+
+  public get windowTitle(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="help-component-title"]')
+  }
+
+  public get searchIcon(): ChainablePromiseElement<
+  Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-Magnifier"]')
+  }
+
+  public get searchPlaceholder(): ChainablePromiseElement<
+  Promise<WebdriverIO.Element>
+  > {
+    return $('[type="search"]')
+  }
+
+  public get topicContent(): ChainablePromiseElement<
+  Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="content"]')
+  }
+  
+  public get articleBackLink(): ChainablePromiseElement<
+  Promise<WebdriverIO.Element>
```

Comment:
```suggestion
    Promise<WebdriverIO.Element>
```
- Prettier does not work in this file.

Reply:
corrected